src/pages/LetterWritePage/components/LetterPaper/LetterReceiverContainer.test.tsx

```diff
@@ -0,0 +1,197 @@
+import { UserEvent } from '@testing-library/user-event';
+import { render, screen, within } from '@/utils/testing-library';
+import { LetterReceiverContainer } from '..';
+import { WriteInputs, writeSchema } from '../..';
+
+const ResizeObserver = vi.fn(() => ({
+  observe: vi.fn(),
+  unobserve: vi.fn(),
+  disconnect: vi.fn(),
+}));
+
+vi.stubGlobal('ResizeObserver', ResizeObserver);
+
+const LetterReceiverContainerComponent = () => {
+  const methods = useForm<WriteInputs>({
+    resolver: zodResolver(writeSchema),
+    defaultValues: {
+      age: [],
+      content: '',
+      gender: '',
+      worryType: '',
+    },
+  });
+  return (
+    <FormProvider {...methods}>
+      <LetterReceiverContainer />,
+    </FormProvider>
+  );
+};
+
+const LetterReceiverSelectedComponent = () => {
+  const methods = useForm<WriteInputs>({
+    resolver: zodResolver(writeSchema),
+    defaultValues: {
+      age: [10, 40],
+      content: '',
+      gender: '나와 같은 성별에게 보내기',
+      worryType: 'LOVE',
+    },
+  });
+  return (
+    <FormProvider {...methods}>
+      <LetterReceiverContainer />,
+    </FormProvider>
+  );
+};
+
+describe('인터랙션 테스트', () => {
+  it('누구에게 보낼까요? 를 누르면 바텀시트가 열린다.', async () => {
+    const { user } = render(<LetterReceiverContainerComponent />);
+
+    const receiverContainer = await screen.findByText('누구에게 보낼까요?', {
+      selector: 'span',
+    });
+
+    await user.click(receiverContainer);
+
+    const bottomSheet = await screen.findByText(
+      '고민을 선택을 하면 나와 비슷한 고민을 가진 낯선 친구에게 전달돼요',
+    );
+    expect(bottomSheet).toBeVisible();
+  });
+});
+
+describe('편지 받는 사람 선택 완료 후, 렌더링 테스트', () => {
+  let user: UserEvent;
+
+  beforeEach(async () => {
+    const { user: userEvent } = render(<LetterReceiverContainerComponent />);
+    user = userEvent;
+
+    const receiverContainer = await screen.findByText('누구에게 보낼까요?', {
+      selector: 'span',
+    });
+
+    await user.click(receiverContainer);
+  });
+
+  it('바텀시트에서 아무것도 선택하지 않은 상황에서 완료 버튼을 누르면, "10~40"이 화면에 보인다.', async () => {
+    const buttonElement = screen.getByRole('button', { name: '완료' });
+    await user.click(buttonElement);
+
+    const age = await screen.findByText('10~40');
+    expect(age).toBeVisible();
+  });
+  it('바텀시트에서 받는 사람 성별 선택 후 완료 버튼을 누르면, "모두에게"가 화면에 보인다.', async () => {
+    const genderChip = screen.getByRole('button', {
+      name: '모두에게 보내기',
+    });
+    await user.click(genderChip);
+
+    const buttonElement = screen.getByRole('button', { name: '완료' });
+    await user.click(buttonElement);
+
+    const gender = await screen.findByText('모두에게');
+    expect(gender).toBeVisible();
+  });
+  it('바텀시트에서 받는 사람 고민 선택 후 완료 버튼을 누르면, "학업"이 화면에 보인다.', async () => {
+    const worryChip = screen.getByRole('button', {
+      name: '학업',
+    });
+    await user.click(worryChip);
+
+    const buttonElement = screen.getByRole('button', { name: '완료' });
+    await user.click(buttonElement);
+
+    const worry = await screen.getAllByText('학업', {
```

Comment:
p5;
getBy 쿼리이니 `await` 은 없어도 괜찮을 것 같아요 😁

```suggestion
    const worry = screen.getAllByText('학업', {
```